### ISSUE TITLE
Website: Example card titles and controls don't handle 400% zoom

### ISSUE THREAD

**Issue by 999-Dev** [microsoft/fluentui#10780]
Accessibility][Reflow : Panel Control] : After resize the browse up to 400%, Text and Controls are not visible.

**User Impact:**
Users who relay on AT will face difficulty if controls and text are not visible after resize the web up to 400%.

**Usage:**
All

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/panel
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999

**Repro Steps:**
1. Open https://developer.microsoft.com/en-us/fabric#/controls/web/panel in Edge.
2. Resize the Web browser  up to 400%.​
3. Refresh the URL.
4. Now navigate by using Tab key and verify the issue.
	
**Actual Result:**
After resize the browser 400% Text and control are not visible.

**Expected Result:**
All controls and text should be visible.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=a37efe7f-9d71-4f3a-b1ff-f93ac9a0c937

**Attachment**
![Text at 100%](https://user-images.githubusercontent.com/55537167/66560702-8402fc80-eb75-11e9-8647-2afb8015c9bc.png)

![After resize the browser 400%](https://user-images.githubusercontent.com/55537167/66560708-87968380-eb75-11e9-8545-719a0b809bba.png)

**Comment by ecraig12345** [comment]
This issue isn't specific to the panel, it applies to the examples for any component. I've updated the title to reflect that.

**Comment by ecraig12345** [comment]
Looks a little different now but still an issue. We should probably make the title always display *above* the controls below a certain screen width. Not sure what to do about the buttons--maybe find a way to make the text shorter and/or hide some of them on small screens?

![image](https://user-images.githubusercontent.com/5864305/118918410-8bf55780-b8e7-11eb-981e-0c39cb702f3f.png)
